src/backend/common/queries/dict_converters/award_converter.py

```diff
@@ -9,6 +9,8 @@
 from backend.common.models.team import Team
 from backend.common.queries.dict_converters.converter_base import ConverterBase
 
+AwardDict = NewType("AwardDict", Dict)
```

Comment:
we can probably TODO these, but we should turn them into `TypedDict`s so we can properly schema-fy them